### ISSUE TITLE
Setup Work For Demo

### ISSUE THREAD

**Issue by datafibers** [datafibers/df_demo#4]
Demo needs to integrate following things
- [x] install dashboard software in the vagrant deb.sh
- [x] create demo setup scripts to run jar, create elastic index, import dashboard, etc

**Comment by SchubertZhu** [comment]
Hi Will,

I have made the changes to convert the data as required. Not sure how to
check it in yet and need your help how to check in the code.

Attached is the code I have changed; search for the method
dataFormatConversion, you will see the changes.

I didn't make any change in client side; I think we only need to change the
input parameters from STREAM_KAFKA to KAFKA_CONNECT_JSON in Agent
application as below:

C:\Temp finance KAFKA_CONNECT_JSON

I tested the code, I can see the input data has been converted correctly in
the method of dataFormatConversion; however, the data in the files have no
changes.

Talk to you next Tuesday.

Have a nice long weekend!

Regards,

Schubert
